packages/themes/bmf/src/components/combobox.scss

```diff
@@ -59,21 +58,21 @@ $visible-options: 5;
 			overflow-x: hidden;
 			box-sizing: border-box;
 			width: 100%;
-			max-height: calc($option-height * $visible-options + 2px);
+			max-height: calc(rem(40) * $visible-options + 2px);
```

Comment:
Ich fand das mit der Variable eigentlich ganz nett, weil es die Berechnung selbsterklärend macht.

Reply:
ist geändert, da dass height Thema für die Option Liste nahezu für alle Themes identisch ist, wäre es eigentlich besser in dem base Theme aufgehoben,